### DISCUSSION TITLE
Raw intensities

### DISCUSSION THREAD

**Discussion by NavitMenahem** [vdemichev/DiaNN#762] [General]
Hello, Thank you for the software. I wanted to ask how can I generate in the Matrices report the value of area under the curve of a protein (called intensity in MaxQuant) and not only the normalized values as LFQ.
I understand that this information exists in the "main report" under Quantity or Normalized but there I didn't understand how to filter for 1% FDR . 
I noticed that there is a command for "no LFQ" does it means that if I use it I will get the intensity instead and if so will the values be filter by an appropriate q-value to 1% FDR?
To sum up, I would really appreciate if you will be able to explain how can I get the initial protein intensity values filtered at 1% FDR.
Thank you
Navit

**Comment by vdemichev** [comment]
Hi Navit,

Can you maybe elaborate in more detail what you mean by 'area under the curve of a protein' and why you need it? The matrices are an easy way to work with the output and they contain protein quantities that are ready to use for the downstream analysis. But when using R or Python, it's recommended to use the main report. Matrices are filtered by default at 1% global protein FDR. For run-specific protein FDR filtering in matrices, can use --matrix-spec-q. Or just use the main report.

Best,
Vadim
